pkg/controller/sriovnetwork/sriovnetwork_controller.go

```diff
@@ -115,6 +116,7 @@ func (r *ReconcileSriovNetwork) Reconcile(request reconcile.Request) (reconcile.
 
 	// Fetch the SriovNetwork instance
 	instance := &sriovnetworkv1.SriovNetwork{}
+	var namespaces = ""
```

Comment:
nit: namespaces -> namespace ?  This looks like will be assigned with the value of the namespace from one certain net-attach-def.

Reply:
fixed